person_class.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = nil
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return false unless of_age? || parent_permission
+  end
+
+  def of_age?
+    return false unless @age >= 18
+  end
```

Comment:
- [ ] Please make the following method `private`, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Private%20method%20of_age%3F) requirement.

- [ ] Kindly note that this method returns `false` if the value of the `@age >= 18` condition is false. This means that if the condition resulted in `true`, it will not return anything. Another format you can write this method with is just simply to return the result of the condition:

```ruby
def of_age?
   `@age >= 18`
end
```